python/lsst/ts/xml/data/sal_interfaces/TunableLaser/TunableLaser_Events.xml

```diff
@@ -36,6 +36,10 @@
     <Subsystem>TunableLaser</Subsystem>
     <EFDB_Topic>TunableLaser_logevent_continuousModeSet</EFDB_Topic>
   </SALEvent>
+  <SALEvent>
+    <Subsystem>TunableLaser</Subsystem>
+    <EFDB_Topic>TunableLaser_logevent_triggerSet</EFDB_Topic>
```

Comment:
Please add a Description (if you keep it).

Also the name sounds like a mode to me -- something that can be set or unset. Please consider `triggered` or `burstTriggered`.